<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/new/prefix1/icons/vkp2.ico</normaloff>:/new/prefix1/icons/vkp2.ico</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>30</y>
     <width>421</width>
     <height>271</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Общие</string>
    </attribute>
    <widget class="QSpinBox" name="spinBox_Text_Song">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="accelerated">
      <bool>false</bool>
     </property>
     <property name="correctionMode">
      <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Скорость бегущей строки</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Показать уведомления</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_Notification">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Сворачивать в трей</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_Hide_Tray">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Download_Dir_BTN">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>200</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Папка загрузки</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Exit_account">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>200</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Выйти из аккаунта</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Цвет окна</string>
    </attribute>
    <widget class="QSlider" name="horizontalSlider_R">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>301</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Red</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_G">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>301</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_B">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>301</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Green</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Blue</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="StandartColor">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>210</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Стандартный</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Random_Color">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>210</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Случайный</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Прозрачность</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSlider" name="transparency_Slider">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>251</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>30</number>
     </property>
     <property name="value">
      <number>99</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Гор. Клавиши</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>281</width>
       <height>175</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideRight</enum>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::DashLine</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>true</bool>
     </property>
     <row>
      <property name="text">
       <string>Плей/Пауза</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Следующая</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Предыдущая</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Добавить в свои аудиозаписи</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Загрузить</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Громкость +</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Громкость -</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Гор. клавиша</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>Alt + Home</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>Alt + PgUp</string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>Alt + PgDn</string>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string>Alt + A</string>
      </property>
     </item>
     <item row="4" column="0">
      <property name="text">
       <string>Alt + D</string>
      </property>
     </item>
     <item row="5" column="0">
      <property name="text">
       <string>Alt + O</string>
      </property>
     </item>
     <item row="6" column="0">
      <property name="text">
       <string>Alt + I</string>
      </property>
     </item>
    </widget>
    <widget class="QTextEdit" name="Text">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>190</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Щелкните дважды по горячей клавише для её замены&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Default_HotKey">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>221</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Сбросить</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Обновление</string>
    </attribute>
    <widget class="QPushButton" name="UpdateOLD">
     <property name="geometry">
      <rect>
       <x>152</x>
       <y>130</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Обновить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CheckNewVER">
     <property name="geometry">
      <rect>
       <x>152</x>
       <y>90</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Проверить </string>
     </property>
    </widget>
    <widget class="QLabel" name="printActualVersion">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>20</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTextEdit" name="WarningUpdate">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>180</y>
       <width>381</width>
       <height>51</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;При нажатии на кнопку &amp;quot;Обновить&amp;quot; программа скачает новую версию и автоматически обновится. Пожалуйста, не закрывайте приложение во время обновления.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>About</string>
    </attribute>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>401</width>
       <height>231</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;VKPlayer 3.0 beta&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;right&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:11pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;right&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:11pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;right&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Разработал : CauMoH&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;right&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;right&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Оставляйте свои вопросы и пожелания по адресам : &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;slanesterov@yandex.ru&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;right&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;caumoh1993@gmail.com&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;right&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;right&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Если Вам понравилось приложение, Вы можете финансово поддержать его разработку. Все подробности на сайте&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;right&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Arial,Helvetica,sans-serif'; font-size:7pt;&quot;&gt;Copyright © 2014 CauMoH&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Open_Site_BTN">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>130</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Перейти на сайт</string>
     </property>
    </widget>
    <widget class="QPushButton" name="aboutQt">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>210</y>
       <width>130</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>About Qt</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="CloseApp">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>6</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Icon">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>4</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>1</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Настройки</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
